deploy/demo/.env.demo.example

```
@@ -0,0 +1,131 @@
+# ---------------------------------------------------------------------------
+# Database (CockroachDB)
+# ---------------------------------------------------------------------------
+
+# [REQUIRED] CockroachDB connection string used by the application.
+# For a single-node demo cluster running alongside Meridian, the
+# cockroachdb service in docker-compose resolves by hostname.
+DATABASE_URL=postgres://root@cockroachdb:26257/defaultdb?sslmode=disable
+
+# [OPTIONAL] GORM database connection pool settings.
+DB_MAX_OPEN_CONNS=25
+DB_MAX_IDLE_CONNS=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_MAX_IDLE_CONNS key should go before the DB_MAX_OPEN_CONNS key

(UnorderedKey)